samples/bluetooth/mesh/sensor_server/README.rst

```diff
@@ -0,0 +1,175 @@
+.. _bluetooth_mesh_sensor_server:
+
+Bluetooth: Mesh Sensor Server
+###############################
+
+The Bluetooth Mesh :ref:`bt_mesh_sensor_srv_readme` sample demonstrates how to set up a basic :ref:`bt_mesh_sensor_srv_readme` model application that provides sensor data to one :ref:`bt_mesh_sensor_cli_readme` model.
+In this sample the on-chip ``TEMP_NRF5`` temperature sensor is used, and ``Button 1`` on the board is used to simulate presence detected.
```

Comment:
Not sure if the correct highlighting/formatting is used here...
Maybe *TEMP_NRF5*, and *Button 1*, if any formatting at all? :)